extension-androidauto/api/extension-androidauto.api

```diff
@@ -71,12 +71,12 @@ public final class com/mapbox/maps/extension/androidauto/MapboxCarMapSurface {
 	public fun toString ()Ljava/lang/String;
 }
 
-public final class com/mapbox/maps/extension/androidauto/widgets/CompassWidget : com/mapbox/maps/renderer/widget/BitmapWidget {
+public final class com/mapbox/maps/extension/androidauto/widgets/CompassWidget : com/mapbox/maps/renderer/widget/RecyclingBitmapWidget {
```

Comment:
```suggestion
public final class com/mapbox/maps/extension/androidauto/widgets/CompassWidget : com/mapbox/maps/renderer/widget/BitmapWidget {
```

This is still a non-backwards compatible change.

the androidauto extension intends to use version `10.9` and "be upgradable to 10.10`

Changing the name of this will make it so the minimum version of androidauto extension is 10.10.

Reply:
Can we make sure to add this to `10.9`?